pkg/proxy/healthcheck/service_health.go

```diff
@@ -137,7 +137,7 @@ func (hcs *server) SyncServices(newServices map[types.NamespacedName]uint16) err
 						UID:       types.UID(nsn.String()),
 					}, nil, api.EventTypeWarning, "FailedToStartServiceHealthcheck", "Listen", msg)
 			}
-			klog.Error(msg)
+			klog.ErrorS(err, msg)
 			continue
```

Comment:
You need to rebuild the 'msg' equivalent on the lifeline, not just log one string